server/events/project_finder.go

```diff
@@ -123,7 +123,7 @@ func (p *DefaultProjectFinder) filterToTerraform(files []string) []string {
 	for _, fileName := range files {
 		// Filter out tfstate files since they usually checked in by accident
 		// and regardless, they don't affect a plan.
-		if !p.isStatefile(fileName) && strings.Contains(fileName, ".tf") {
+		if !p.isStatefile(fileName) && (strings.Contains(fileName, ".tf") || fileName == "terragrunt.hcl") {
```

Comment:
```suggestion
		if !p.isStatefile(fileName) && (strings.Contains(fileName, ".tf") || filepath.Base(fileName) == "terragrunt.hcl") {
```

`fileName` is a full file path here. If you try this with a directory prefix it will fail.